### DISCUSSION TITLE
Adding a new color in the element-plus color scheme

### DISCUSSION THREAD

**Discussion by maximilien-leleu** [element-plus/element-plus#12285] [Q&A]
Hi @YunYouJun 
I've seen you are a master in the element-plus community. I'm trying to override the element-plus color scheme but I need to add an additionnal color to it.
[github.com/realite-digita](https://github.com/realite-digitale/element-plus-scss-issue/blob/main/src/styles/element/index.scss)…
Thanks for helping

**Comment by YunYouJun** [comment]
You mean you want to add a new semantic color, like 'amazing', right?
Unfortunately, we can't do that yet, because we explicitly iterate through the specified semantics.

https://github.com/element-plus/element-plus/blob/5edc3a3adf17cd8b4546df47840b65e73f194312/packages/theme-chalk/src/var.scss#L18

**Comment by YunYouJun** [comment]
If you want to do this, you can refer to `src/var.scss` to do your build operation.

Of course, I think your idea is worth considering, and maybe we can improve the generation of element-plus.
But to avoid the invisible problems caused by such a transformation, we need to see what other partners propose.

cc @element-plus/backers
